webapp/components/LoginForm/LoginForm.vue

```diff
@@ -100,21 +83,16 @@ export default {
 </script>
 
 <style lang="scss">
-.login-image {
-  width: 90%;
-  max-width: 200px;
-}
-.login-card {
-  position: relative;
+.login-form {
+  width: 80vw;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-12T13:12:31Z" title="Wednesday, February 12th 2020, 2:12:31 pm +01:00">Feb 12, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -100,21 +83,16 @@ export default {
 </script>
 
 <style lang="scss">
-.login-image {
-  width: 90%;
-  max-width: 200px;
-}
-.login-card {
-  position: relative;
+.login-form {
+  width: 80vw;
```


I had to google this... this set it to 80% of the viewport, not 80% of the parent container, is that correct? these can be different if the user's viewport is smaller than the parent container, is that right?
I saw this as a reference https://stackoverflow.com/questions/31039979/css-units-what-is-the-difference-between-vh-vw-and

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-17T09:14:40Z" title="Monday, February 17th 2020, 10:14:40 am +01:00">Feb 17, 2020</time>_

---


`vh` and `vw` always reference the viewport height and width, yes. So basically the size of the browser window.

It can be both bigger or smaller than the parent – sometimes the parent overflows the viewport (like on the index page, where you need to scroll down to see all the content), sometimes the parent only takes up part of the viewport (like on the settings pages, where you have the side bar and the actual content only takes up about 2/3 of the screen).

I like to use these values in media queries and usually only on some sort of `wrapper` – so I wouldn't use it on elements like titles or images or any other content. In this case the `login-form` is the only element on the page and I thought it would be more explicit to say: "I want this element to take up 80% of the width of the screen"  than "I want this element to be 80% as wide as some obscure layout element it is nested in".